src/main/kotlin/org/llamarama/petty/blocks/CageBlock.kt

```diff
@@ -0,0 +1,53 @@
+package org.llamarama.petty.blocks
+
+import net.minecraft.block.BlockState
+import net.minecraft.block.ShapeContext
+import net.minecraft.block.TallPlantBlock
+import net.minecraft.block.enums.DoubleBlockHalf
+import net.minecraft.entity.Entity
+import net.minecraft.entity.passive.ParrotEntity
+import net.minecraft.state.property.EnumProperty
+import net.minecraft.state.property.Properties
+import net.minecraft.util.math.BlockPos
+import net.minecraft.util.shape.VoxelShape
+import net.minecraft.util.shape.VoxelShapes
+import net.minecraft.world.BlockView
+import net.minecraft.world.World
+
+open class CageBlock(settings: Settings?) : TallPlantBlock(settings) {
```

Comment:
I forgot this, why is it an *open* class?

Reply:
So that it isn't final if we ever want to add other cage-type blocks with the same collison box.

Reply:
ah okay, wouldn't it make sense to "close" it for now and open it once we need it?

Reply:
Yeah that would probably be better.